benches/isolated_buffering.rs

```diff
@@ -30,16 +28,11 @@ impl Sink for NullSink {
 }
 
 fn benchmark_buffers(c: &mut Criterion) {
-    let num_lines: usize = 100_000;
+    let num_lines: usize = 10_000;
```

Comment:
How would you feel about reducing this batch size? It's a lot of events since criterion will run this in a loop with warmup.